test/acceptance/test/testsuite_common_test.go

```diff
@@ -16,7 +16,7 @@ func GomegaFail(message string, callerSkip ...int) {
 
 	//Show pods
 	err := showItems("pods")
-	if err == nil {
+	if err != nil {
 		log.Infof("Failed to print the pods")
 	}
```

Comment:
This is super minor but wouldnt this function be more helpful if it returned an error instead of just logging it.

Reply:
Sure thing but couldn't think of a use case where the caller would interested to do much with the error, so thought just log the error and done with it.